docs/file_formats/fip.md

```diff
@@ -53,9 +75,11 @@ Each fiber photometry session will primarily be analyzed by using the average si
 * `CameraFrameNumber` Frame counter given by the camera API
 * `CameraFrameTime` Frame acquisition time given by the camera API
 * `Background` CMOS dark count floor signal
-* `Fiber_0` Average signal values for Fiber_0's selected ROI
-* `<...>` (Variable number of columns)
-* `Fiber_N` Average signal values for Fiber_N's selected ROI.
+* `Fiber_0`, `Fiber_1`, …, `Fiber_{K-1}` Average signal values from patch_cord_k's ROI (one column per patch cord/ROI)
+
+Note: There are always exactly K `Fiber_n` columns (n = 0 … K-1) corresponding to the configured patch cords and ROIs for the rig. If N < K (where N is the number of implanted fibers), then patch cords N through K-1 are unused and their corresponding columns contain data from unconnected patch cords.
```

Comment:
As I mentioned in #58, if anything we should be simplifying this by removing the notion of Fiber and using ROI instead. I kept it for backwards compatibility with Kenta's CSV. Otherwise, given what I mentioned above, this description just further complicates the life of anyone trying to understand the file standard and respective implementation.